nevergrad/optimization/optimizerlib.py

```diff
@@ -502,6 +515,8 @@ class ParametrizedCMA(base.ConfiguredOptimizer):
     inopts: optional dict
         use this to averride any inopts parameter of the wrapped CMA optimizer
         (see https://github.com/CMA-ES/pycma)
+    use_cma_bounds: bool
+        use CMA own bounding option (if the parametrization is fully bounded)
```

Comment:
```suggestion
        use CMA own bounding option (if the parametrization is fully bounded)
        instead of relying on the parametrization system.
```